src/Agent/NewRelic/Agent/Core/Configuration/ReportedConfiguration.cs

```diff
@@ -210,10 +211,18 @@ public ReportedConfiguration(IConfiguration configuration)
         public bool DistributedTracingEnabled => _configuration.DistributedTracingEnabled;
 
         [JsonProperty("distributed_tracing.sampler.root.sampler_type")]
+        public string RootSamplerName => RootSamplerType.ToSamplerTypeString();
+        [JsonIgnore]
         public SamplerType RootSamplerType => _configuration.RootSamplerType;
+
         [JsonProperty("distributed_tracing.sampler.remote_parent_sampled.sampler_type")]
+        public string RemoteParentSampledSamplerName => RemoteParentSampledSamplerType.ToSamplerTypeString();
```

Comment:
Sorry for the noise in this file; this is where the relevant changes start. Here to line 225